index.html

```diff
@@ -487,6 +482,7 @@
 </html>
 
 <!-- preserve before running Tidy -->
+<!--
```

Comment:
I'm not sure I understand the purpose of this section... Is it for auto-linking?